leveldb-benchmark/src/main/java/org/iq80/leveldb/benchmark/DbBenchmark.java

```diff
@@ -485,15 +485,18 @@ private void readReverse(ThreadState thread)
     private void readRandom(ThreadState thread)
     {
         int found = 0;
+        long bytes = 0;
         for (int i = 0; i < reads; i++) {
             byte[] key = formatNumber(thread.rand.nextInt(num));
             byte[] value = db.get(key);
             if (value != null) {
-                found++;
+                found ++;
+                bytes += key.length + value.length;
+                thread.stats.finishedSingleOp();
```

Comment:
Operation should be counted as done event if the key is not found. This will give incorrect results.